plugins/3MFReader/WorkspaceSection.qml

```diff
@@ -15,7 +15,11 @@ Item
     property Component content: Item { visible: false  }
 
     property alias comboboxTitle: comboboxLabel.text
-    property Component combobox: Item { visible: false }
+    property Component combobox: Item
+        {
+            z: -1
```

Comment:
this isn't really needed. The placeholder is an invisible `Item` conaining no style, and no children. If it's not rendered anyway no need to place it on top

Reply:
[eac2d3b](https://github.com/Ultimaker/Cura/pull/18603/commits/eac2d3b173e0819bddd8fd68e132932e9ad5617e)